include/deal.II/base/vectorization.h

```diff
@@ -2707,6 +2707,25 @@ vectorized_transpose_and_store(const bool                    add_into,
 
 
 /**
+ * Relational operator == for VectorizedArray
+ *
+ * @relates VectorizedArray
+ */
+template <typename Number>
+inline DEAL_II_ALWAYS_INLINE
+bool
+operator == (const VectorizedArray<Number> &lhs,
```

Comment:
I had a brief look at the `matrix_free` headers, and nothing looks out of place to me (`#include <deal.II/base/vectorization.h>` is an indirect inclusion in `mapping_info.templates.h`). The line `mapping_info.templates.h:511` is inside `namespace internal`, and there are some direct qualifications of classes in the the `dealii` within this section of code (e.g. `dealii::Triangulation`), but it seems to have never been necessary to explicitly use `dealii::VectorizedArray` itself. 

In summary, I'm out of ideas as to what the cause of this could have been. Do you have any specific insights as to what I should be looking for, or can we consider this fixed?